src/components/compounds/Hero/index.js

```diff
@@ -1,14 +1,75 @@
-import React from 'react';
-
+import React, { Component } from 'react';
 import Button from 'Components/atoms/Button';
 import './style.scss';
+import { connect } from 'react-redux';
+import Signup from '../Signup/Signup';
+import Modal from '../../atoms/Modal';
+import { showModal, closeModal, userFail } from '../../../store/actions/authUser';
+
+/*eslint-disable*/
+class Hero extends Component {
+  state={
+    modalOpen: false
+  }
+
+  /**
+   * @memberOf Signup
+   * @method showModalHandler
+   * @param {*} event
+   * @return {*} setstate
+   */
+  showModalHandler = () => {
+    const { showModalHandle } = this.props;
+    showModalHandle();
+  }
+
+  /**
+   * @memberOf Signup
+   * @method closeModalHandler
+   * @param {*} event
+   * @return {*} setstate
+   */
+  closeModalHandler = () => {
+    const { closeModalHandle, userfail } = this.props;
+    closeModalHandle();
+    userfail();
+  }
+
+  render() {
+    const {
+      modalOpen,
+      isAuth
+    } = this.props;
+
+    return (
+      <div className="hero">
+        <h1 className="title">AUTHOR&apos;S HAVEN</h1>
+        <p className="text">A community where readers and writers come together to share and discuss knowledge and ideas.</p>
+        {isAuth
+          ? <Button text="WRITE" onClick={() => toastr.success('comming soon...')} />
+          : <Button text="GET STARTED" onClick={this.showModalHandler} />
```

Comment:
We can re-write this like,
```suggestion
          : <Button text="GET STARTED" onClick={this.props.showModalHandler} />
```